app/Models/User.php

```diff
@@ -123,20 +123,6 @@ public function setTzAttribute($value)
         $this->attributes['timezone'] = $value;
     }
 
-    /**
-     * Return a File model
-     */
-    public function getAvatarAttribute()
```

Comment:
I think we still need this so we can call `$user->avatar->url`, which will form the correct URL (look at `Models/File.php`)

Reply:
Let’s me check again to make other commit please

Reply:
No problem. It's just so the URL is formed correctly and if they want to use the Laravel file storage going to a different place.

Reply:
I didn't see File model, so that logic didn't make me sense, but now I have corrected these lines.